packages/react/src/components/Tile/Tile.js

```diff
@@ -485,15 +489,24 @@ export class ExpandableTile extends Component {
         : this.state.tileMaxHeight + this.state.tilePadding,
     };
     const content = this.getChildren().map((child, index) => {
-      return React.cloneElement(child, { ref: index });
+      // Changes aria-hidden above/below the fold content based on the state of expanded so the content doesn't get announced by screenreaders at the wrong time.
+      const isHidden = index === 0 ? expanded : !expanded;
+
+      return React.cloneElement(child, {
+        ref: index,
+        'aria-hidden': isHidden,
+      });
     });
-    const buttonId = this.props.id ? `${this.props.id}__button` : this.uid;
+
+    const tileContentId = `tile-content-a11y-${this.instanceId}`;
```

Comment:
For naming conventions, I think you can just use `tile-content`, no need for `a11y` (or we can change so that `a11y` is the convention, I'm just not sure if that's used elsewhere in the codebase.